source/core/tests/silver/domain/validations/test_enum_validations.py

```diff
@@ -73,7 +74,27 @@ def test__quality_enum_validations_when_points_column_is_null__then_data_is_inva
     spark: SparkSession,
 ) -> None:
     # Arrange
-    unpacked_submitted_transactions = UnpackedSubmittedTransactionsBuilder(spark).build()
+    testschema = StructType(
+        [
+            StructField("id", StringType(), False),
+            StructField(
+                "points",
+                ArrayType(
+                    StructType(
+                        [
+                            StructField("quality", StringType(), True),
+                        ]
+                    ),
+                    True,
+                ),
+                True,
+            ),
+        ]
+    )
+
+    data = [("124", None)]
+
+    unpacked_submitted_transactions = spark.createDataFrame(data, schema=testschema)
```

Comment:
```suggestion
unpacked_submitted_transactions = UnpackedSubmittedTransactionsBuilder(spark).add_row(points=None).build()
```

wouldn't this be possible?